OneDrive-Cloud-Player/Exceptions/ServiceNotInitializedException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
```

Comment:
I probably wouldn't have added a user-defined exception class which is used once. If this is later needed, we can always add it.

Reply:
I added that class to make it very clear why the service fails. I see no reason why it would be a requirement to have multiple uses before a custom exception should be considered.

Reply:
But I don't mind using a more generic exception if you insist on using that over a custom one.

Reply:
Nah it's fine